torch/nn/init.py

```diff
@@ -0,0 +1,245 @@
+from functools import reduce
+from operator import mul
+import math
+import random
+
+import torch
+from torch.autograd import Variable
+
+
+def uniform(tensor, a=0, b=1):
+    """Fills the input Tensor or Variable with values drawn from a uniform U(a,b)
+
+    Args:
+        tensor: a n-dimension torch.Tensor
+        a: the lower bound of the uniform distribution
+        b: the upper bound of the uniform distribution
+
+    Examples:
+        >>> w = torch.Tensor(3, 5)
+        >>> nninit.uniform(w)
+    """
+    if isinstance(tensor, Variable):
+        uniform(tensor.data, a=a, b=b)
+        return tensor
+    else:
```

Comment:
nit: you could remove the `else:` and move the actual implementation one indentation level back. I think that'd look nicer.